### DISCUSSION TITLE
Background sync iOS behaviour

### DISCUSSION THREAD

**Discussion by SimonPickup** [MobiusSync/MobiusSync#30] [Q&A]
The iOS scheduling of background sync sometimes seems to have a mind of its own.

Opening the app more often may encourage iOS to schedule more background sync (and influence the timing of quick syncs at least) as it is intended to be preloading content to be later viewed when the app is opened.

On my devices, I can get 1-2h total sync time per 24, sometimes over 2h, with the default timing settings.

Please add your own experiences, recommended settings or tips below.

**Comment by cbeams** [comment]
Thanks for setting this up, @sjpickup. Rather than repeat it all here, I'll link to the issue comment thread that led to opening this discussion:

https://github.com/MobiusSync/MobiusSync/issues/7#issuecomment-770217846

In summary: if you're working with an iOS device whose battery is no longer able to hold much of a charge, you may end up waiting a long time (a full day in my case) for first background sync to occur, and with irregular / sporadic syncs thereafter.

**Comment by louwers** [comment]
Not seeing any syncs on a new 13 Pro with a full battery, unless if I open the app.

**Comment by alx-xlx** [comment]
I run Mobius sync on ipad and it has never background synced ever. Perhaps if there could be a shortcut widgets for mobius sync to start the app and automatically "Rescan all"
